Dockerfile

```diff
@@ -7,12 +7,17 @@ COPY documentation/examples/prometheus.yml  /etc/prometheus/prometheus.yml
 COPY console_libraries/                     /usr/share/prometheus/console_libraries/
 COPY consoles/                              /usr/share/prometheus/consoles/
 
-RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/
+RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/ && \
+    addgroup -g 1000 prometheus && \
+    adduser -s /bin/sh -S -H -u 1000 -G prometheus prometheus && \
```

Comment:
Hopefully the following will address all your comments. 

1. I will remove `-S` as it's not a system account, and use the `-D`.
2. I don't think there is a need for a home directory,  specially if this is to be switch to user nobody.
3. `1000` was picked as there is less chance for uid collision with the host system should you decide to share a volume. At the end of the day it's arbitrary and it becomes a mute point should it be decided to use the `nobody` user.
4. The `addgroup` and `adduser` become mute should you decide to use the `nobody` user.